Ports/chibi-scheme/package.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='chibi-scheme'
+version='0.10'
+useconfigure='true'
+files=(
+    "https://github.com/ashinn/chibi-scheme/archive/refs/tags/${version}.tar.gz ae1d2057138b7f438f01bfb1e072799105faeea1de0ab3cc10860adf373993b3"
+)
+localexpanddir="$(pwd)/build-local"
+localbuilddir="$localexpanddir/chibi-scheme-$version"
+
+export LD_LIBRARY_PATH=$localbuilddir/usr/local/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
+export DYLD_LIBRARY_PATH=$localbuilddir/usr/local/lib${DYLD_LIBRARY_PATH:+:$DYLD_LIBRARY_PATH}
+
+configure() {
+    mkdir -p $localbuilddir
+}
+
+build() {
+    # make local version
+    run make CC="$HOST_CC" PLATFORM=serenity DESTDIR=$localbuilddir
+    run make DESTDIR=$localbuilddir install
+    run make clean
+    run make PLATFORM=serenity CHIBI=$localbuilddir/usr/local/bin/chibi-scheme
```

Comment:
This looks like it might be a case for `host_env`?

```suggestion
    host_env

    run make PLATFORM=serenity DESTDIR=$localbuilddir
    run make DESTDIR=$localbuilddir install
    run make clean

    target_env

    run make PLATFORM=serenity CHIBI=$localbuilddir/usr/local/bin/chibi-scheme
```

Reply:
I did not know about host_env/target_env, thanks! I'll give that a try.

Reply:
Updated to use `host_env`/`target_env`.